edb/edgeql/compiler/inference/volatility.py

```diff
@@ -0,0 +1,316 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2008-present MagicStack Inc. and the EdgeDB authors.
```

Comment:
This probably should be 2020 and not 2008. I wonder how much does this matter, though in terms of any real consequences. I assume it matters a little bit, otherwise why would we even bother, so it probably needs to be at least somewhat accurate.